Backpack/Icon/Classes/BPKSmallIconView.m

```diff
@@ -0,0 +1,35 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2018-2021 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+#import "BPKSmallIconView.h"
+
+NS_ASSUME_NONNULL_BEGIN
+
+@implementation BPKSmallIconView
+
+- (instancetype)initWithIconName:(nullable BPKSmallIconName)iconName {
```

Comment:
Why is the parameter `nullable` here? I know it's because it's `nullable` in the `BPKIconView` init. Why is it  `nullable` there tho 🤔

Reply:
`setIconName` also accepts a nullable value, and renders nothing if `nil` is provided 🤔
Seems like this was a deliberate choice at some point.

Reply:
Fair enough :)